resource-managers/yarn/src/main/resources/META-INF/services/org.apache.spark.deploy.security.ServiceCredentialProvider

```diff
@@ -0,0 +1,3 @@
+org.apache.spark.deploy.yarn.security.YARNHadoopFSCredentialProvider
```

Comment:
> move the core META-INF to Mesos

That's not enough. Because the mesos and yarn jar will both still exist, so both files will still be loaded.